README.md

```diff
@@ -146,13 +153,16 @@ SELECT c.customer_id, c.source, c.region, c.member_since,
 
 ### Defining dependencies
 
-As these SQL files are related to Airflow DAGs, there is an expectation that there should be dependencies so you can break up your SQL into
-multiple reproduceable steps. We offer two ways to define dependencies within an `astro` sql file.
+When running SQL queries in Airflow DAGs, you need to define dependencies that break up your SQL into
+multiple, reproduceable steps. We offer two ways to define dependencies within an `astro` SQL file:
 
-The first way to define a dependency is by defining a variable via the `template_vars` option.
+The first way to define a dependency is by defining a variable via the `template_vars` frontmatter argument. We set `template_vars` so that our `join_customers_and_orders` query cannot run until both our `customers_table` and `agg_orders` queries finish. This data dependecy is equivalent task dependency in our DAG. The only difference is that we're defining it directly in our SQL instead of using Airflow's dependency operators. 
```

Comment:
```suggestion
The first way to define a dependency is by defining a variable via the `template_vars` frontmatter argument. We set `template_vars` so that our `join_customers_and_orders` query cannot run until both our `customers_table` and `agg_orders` queries finish. This data dependency is equivalent to a task dependency in our DAG. The only difference is that we're defining it directly in our SQL instead of using Airflow's dependency operators. 
```